src/tests/ftest/control/version.py

```diff
@@ -118,5 +118,6 @@ def test_version(self):
                     tool, version, rpm_version)
                 errors.append(msg)
 
-        if errors:
-            self.fail("\n---- Errors detected! ----\n{}".format("\n".join(errors)))
+        self.log.info("###### Errors ######")
```

Comment:
It may be a bit misleading to log a message containing the text `Errors` even when they may not be any.  Maybe this would be more appropriate:
```suggestion
        self.log.info("###### Test Result ######")
```

Reply:
Done